src/rest/Query.ts

```diff
@@ -5,6 +5,13 @@ import { Context } from '../ui/PipelineContext/PipelineGlobalExports';
 import { ICategoryFacetRequest } from './CategoryFacetRequest';
 import { IFacetRequest } from './Facet/FacetRequest';
 
+export interface IFacetOptions {
```

Comment:
We should have a description for that interface if we want it to be exposed in the public documentation. Suggestion:

> The available global configuration options when requesting facets through the [`facets`]{IQuery.facets} array.